tests/config.vapi.in

```diff
@@ -0,0 +1,14 @@
+[CCode (cheader_filename = "config.h")]
```

Comment:
```suggestion
[CCode (cheader_filename = "config.h", lower_case_cprefix = "")]
```
It will save you all the CCode attributes below :)

Reply:
Wow that was cool